validator/src/main/java/com/google/daq/mqtt/validator/Validator.java

```diff
@@ -538,6 +534,11 @@ private void sendValidationResult(Map<String, String> origAttributes, Object mes
       ReportingDevice reportingDevice) {
     try {
       ValidationEvent validationEvent = makeValidationEvent();
+      validationEvent.status = reportingDevice.getErrorStatus();
```

Comment:
Should makeValidationEvent() help with some of the building below?

Reply:
The function creates the basic common bits, but then it's filled in quite
differently for the different uses -- I'm all for generalization, but not
quite sure how to do it in this case!

Reply:
ok